test/distributed/_tensor/test_dtensor_compile.py

```diff
@@ -469,7 +480,7 @@ def forward(self, input):
             FileCheck().check(
                 "buf0 = torch.ops._c10d_functional.all_gather_into_tensor.default(primal"
             ).check("buf1 = torch.ops._c10d_functional.wait_tensor.default(buf0").check(
-                "extern_kernels.mm(buf0,"
+                "extern_kernels.mm(buf1,"
```

Comment:
Was this test broken? Curious since I don't think your test affects this test.

Reply:
Hmm I don't think it was. `wait()` was just broken in some situations, like when there is a view right after it, but this PR technically changed the name of the buffer of all downstream users of `wait_tensor` (so use the output of `wait_tensor` instead of the input)